direct/io-bulk-fs/src/main/java/cz/o2/proxima/direct/bulk/AbstractBulkFileSystemAttributeWriter.java

```diff
@@ -82,6 +115,7 @@ void write(StreamElement data, CommitCallback commit, long watermark, long seqNo
 
   private long seqNo = 0;
   private transient Map<Long, Bulk> writers = null;
+  private transient @Nullable LateBulk lateBulk = null;
```

Comment:
As we discuss on slack: here can be more than one lateBulk in case of multiple timeframes (months)

Reply:
Fixed.